yb-voyager/cmd/exportDataFromTarget.go

```diff
@@ -92,7 +92,7 @@ Therefore, we allow users to override the ssl root cert in export-data-from-targ
 For logical-replication connecter, all SSL modes are supported.
 */
 func verifySSLFlags(cmd *cobra.Command, msr *metadb.MigrationStatusRecord) error {
-	allowedSSLModes := []string{constants.DISABLE, constants.PREFER, constants.ALLOW, constants.REQUIRE, constants.VERIFY_CA, constants.VERIFY_FULL}
+	allowedSSLModes := AllSSLModes
```

Comment:
would it make more sense to use `ValidSSLModesForSourceDB[YUGABYTEDB]` ?

Reply:
hmm, probably. let me take a look.